tests/components/jvc_projector/test_coordinator.py

```diff
@@ -29,7 +28,7 @@ async def test_coordinator_update(
     )
     await hass.async_block_till_done()
     assert mock_device.get_state.call_count == 3
-    coordinator = hass.data[DOMAIN][mock_integration.entry_id]
+    coordinator = mock_integration.runtime_data
```

Comment:
Ideally this should be refactored not to touch the internals

Reply:
yep I can do that. I made this PR as a precursor to adding new functionality so I can update those tests as well